src/NativeShell/src/build.gradle

```diff
@@ -0,0 +1,11 @@
+allprojects {
+    repositories {
+        maven {
+            url 'https://jitpack.io'
+        }
+    }
+}
+
+dependencies {
+    compile 'com.github.dkanada:jellyfin-apiclient-java:a7e42b7826'
```

Comment:
What's this namespace or whatever it is?

Reply:
I will update this with the official repo as soon as my apiclient changes get merged. I was using my fork for testing.